.circleci/config.yml

```diff
@@ -305,13 +315,22 @@ workflows:
   smoketest-staging-workflow:
     triggers:
       - schedule:
-          # This test is run on the 20th minute of the hour, staggered separately from the
-          # smoke tests in the identity-monitor repo
-          # https://github.com/18F/identity-monitor/blob/master/.circleci/config.yml
           cron: "20 * * * *"
           filters:
             branches:
               only:
                 - stages/staging
     jobs:
       - smoketest-staging
+
+  # PROD
+  smoketest-prod-workflow:
+    triggers:
+      - schedule:
+          cron: "30 * * * *"
```

Comment:
Once this PR has been merged, the :30 slot will be free: https://github.com/18F/identity-monitor/pull/89